tests/tests.py

```diff
@@ -407,6 +449,158 @@ def test_add_remove_attachment(self):
 
 
 
+class EntryHistoryTests3(KDBX3Tests):
+
+    #---------- History -----------
+
+    def test_find_history_entries(self):
+        '''run some tests on entries created by pykeepass'''
+        prefix = 'TFE_'
+        postfix = '_tmpA'
+        changed = 'tfe_changed_'
+        # create a new copy of the file to ensure the original file remains unchanged
+        a = self.kp.filename.split('.')
+        fn = '.'.join(('.'.join(a[:-1]) + postfix, a[-1]))
+        keyfile = os.path.join(base_dir, self.keyfile)
+        self.kp.save(filename=fn)
```

Comment:
Why is it necessary to make a new database for these tests?  Can't you use the existing test database and just not save it?

Reply:
I noticed the for entries in the test database the "History" element is always there but sometimes empty but for entries created by pykeepass the "History" element is missing for entries with no history. My first assumption was that the kp.save() function applies magic to add the "History" element when missing. Therefore I decided to save the DB and reload it.
Now that we know that there's no magic, we can skip this test